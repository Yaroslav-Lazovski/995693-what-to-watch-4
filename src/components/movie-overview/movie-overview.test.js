import React from "react";
import renderer from "react-test-renderer";

import MovieOverview from "./movie-overview.jsx";


const movie = {
  director: `Movie director`,
  starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
  ratingScore: 6.4,
  ratingCount: 240,
  description: `Movie description`
};


describe(`MovieOverview Snapshot`, () => {
  it(`Should MovieOverview render correctly`, () => {
    const tree = renderer
      .create(<MovieOverview
        director= {movie.director}
        starring= {movie.starring}
        ratingScore={movie.ratingScore}
        ratingCount={movie.ratingCount}
        description={movie.description}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
