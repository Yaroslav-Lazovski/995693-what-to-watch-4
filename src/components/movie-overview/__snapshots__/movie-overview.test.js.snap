// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieOverview Snapshot Should MovieOverview render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      6,4
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        240
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Movie description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Movie director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Movie starring
         and other
      </strong>
    </p>
  </div>,
]
`;
